droidlet/dashboard/web/src/StateManager.js

```diff
@@ -67,7 +66,13 @@ class StateManager {
     this.memory = this.initialMemoryState;
     this.processRGB = this.processRGB.bind(this);
     this.processDepth = this.processDepth.bind(this);
+    this.processRGBDepth = this.processRGBDepth.bind(this);
```

Comment:
why do we need all of processRGBDepth, processDepth and processRGB?

Reply:
we can get rid of `processRGBDepth`, I just thought it would be a natural thing for an RGBD object to be logged.

If you look at the implementations, `processRGBDepth` just calls `processRGB` and `processDepth`